music21/setup.py

```diff
@@ -0,0 +1,75 @@
+##!/usr/bin/env python
```

Comment:
This file seems to have been copied and moved accidentally -- can it be deleted?